docs/excel/custom-functions-debugging.md

```diff
@@ -83,32 +81,35 @@ You can use Visual Studio Code to debug custom functions in Excel Online in the
 ### Run your add-in from Visual Studio Code
 
 1. Open your custom functions root project folder in [VS Code](https://code.visualstudio.com/).
-2. In VS Code choose **Terminal -> New Terminal**.
-3. Choose **Terminal -> Run Task** and type or select **Watch**. This will monitor and rebuild for any file changes.
-4. Choose **Terminal -> Run Task** and type or select **Dev Server**. 
+2. Choose **Terminal -> Run Task** and type or select **Watch**. This will monitor and rebuild for any file changes.
+3. Choose **Terminal -> Run Task** and type or select **Dev Server**. 
 
 ### Start the VS Code debugger
 
 4. Choose **View -> Debug** or enter **Ctrl+Shift+D** to switch to debug view.
 5. From the Debug options, choose **Office Online (Edge)**.
 6. Open Excel Online using the Microsoft Edge browser, open Excel Online, and create a new workbook.
 7. Choose **Share** in the ribbon and copy the link for the URL for this new workbook.
+8. Select **F5** (or choose **Debug -> Start Debugging** from the menu) to begin debugging. A prompt will appear, which asks for the URL of your document.
```

Comment:
```suggestion
8. Select **F5** (or choose **Debug> Start Debugging** from the menu) to begin debugging. A prompt will appear, which asks for the URL of your document.
```

Reply:
Shouldn't this use ` -> ` like all of the other places?